cmake/modules/Finduring.cmake

```diff
@@ -0,0 +1,21 @@
+# - Find
+#
+# URING_INCLUDE_DIR - Where to find liburing.h
+# URING_LIBRARIES - List of libraries when using uring.
+# URING_FOUND - True if uring found.
+
+find_path(URING_INCLUDE_DIR liburing.h)
+find_library(URING_LIBRARIES liburing.a uring)
```

Comment:
why do you add `.a` in the library name? and why do you specify a path of "uring" for it?

Reply:
@tchaikov 
I use the static library to avoid error when building ceph-bluestore-tool.